data_entry/art_card.md

```diff
@@ -89,6 +89,23 @@ The default configuration restricts the maxiumum number of patients who can be s
 
 5. Click the "Save" button in the bottom right hand corner 
 
+###Determine the number of patients scheduled on a perticular.
```

Comment:
@solemabrothers 

1. Add a screenshot with the error 

2. Change the title to "Unable to determine the number of patients scheduled due to error on the ART Clinical Assessment form"

Reply:
Done